### PR TITLE
Seeding

### PR DESCRIPTION
**Pull request by drtcxrch**
@jsmithb117 Please have a look at the file called seeder.js in the DB folder. Thanks!

### CODE REVIEW

File: db/seeder.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  var month = dateParts[1];
+  var day = dateParts[2][0] === '0' ? dateParts[2].slice(1) : dateParts[2];
+  var year = dateParts[3];
+
+  return month + ' ' + day + ' ' + year;
+
+};
+
+let saveReviews = (course) => {
+  // console.log('called');
```
Comment (jsmithb117) [inline db/seeder.js:47]: Stray leftover console.log?

File: db/seeder.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      review.reviewText = faker.lorem.paragraph();
+      reviews.push(review);
+    }
+
+    return reviews;
+  };
+
+  var reviews = new Reviews({
+    courseNumber: course,
+    Reviews: reviewsGenerator(reviewCount)
```
Comment (jsmithb117) [inline db/seeder.js:78]: Is reviews supposed to be capitalized on line 78?

**Review by jsmithb117 — COMMENTED** [review]
Very nice!

File: db/seeder.js (hunk 1 above)
Comment (drtcxrch) [reply db/seeder.js]: Yep! I got rid of it.

File: db/seeder.js (hunk 2 above)
Comment (drtcxrch) [reply db/seeder.js]: Good eye! Fixed!

**Pull request merged**